Simplenote/Classes/UITextView+Simplenote.swift

```diff
@@ -34,3 +60,56 @@ extension UITextView {
         return attributedText.attribute(.attachment, at: index, effectiveRange: nil) as? T
     }
 }
+
+
+// MARK: - Geometry
+//
+extension UITextView {
+
+    /// Returns the Bounding Rect for the specified `Range<String.Index>`
+    ///
+    func boundingRect(for range: Range<String.Index>) -> CGRect {
+        let nsRange = text.utf16NSRange(from: range)
```

Comment:
I believe `NSRange(range, in: text)` should also work!

Reply:
You're a genius, looks SO MUCH better!!